CHANGELOG.rst

```diff
@@ -1,14 +1,29 @@
-Latest
+0.8.2
+=====
+
+New Features
+------------
+
+Bug Fixes
+---------
+
+- Shows a warning if observation time is missing [#2293]
+- Updates MapCube to access the correct properties of the namedtuple SpatialPair [#2297]
+
+0.8.1
 ======
 
-New features
+New Features
```

Comment:
I know this is not about this PR, but surely the 'new features' section is redundant in a bug fix?